tests-beta/android/MASVS-CRYPTO/MASTG-TEST-0211.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Weak Encryption
+platform: android
+id: MASTG-TEST-0211
+type: [static]
+weakness: MASWE-0020
+---
+
+## Overview
+
+In this test case, we will look for the use of weak encryption in Android applications. To do this, we need to focus on the cryptographic implementations of algorithm such as DES, 3DES,Weak encryption modes (e.g. ECB) and Cipher.getInstance("AES") defaults to ECB.
```

Comment:
This needs to be more elaborate, similar to  https://mas.owasp.org/MASTG/tests-beta/android/MASVS-CRYPTO/MASTG-TEST-0208/

Be careful with missing whitespace and ensure code elements are enclosed in backticks, e.g., `Cipher`

Also, try to use the `refs` we included in MASWE-0020 wherever possible as inline links for the text. Especially the ones from Support Google and Developer Android.